R/fwrite.R

```diff
@@ -84,11 +84,11 @@ fwrite = function(x, file="", append=FALSE, quote="auto",
     if (file.exists(file)) {
       suggested <- if (append) "" else gettextf("\nIf you intended to overwrite the file at %s with an empty one, please use file.remove first.", file)
       warningf("Input has no columns; doing nothing.%s", suggested)
-      return(invisible())
+      return(invisible(file))
```

Comment:
Maybe here `NULL` should be returned, since the actual `file` is not affected at all by `fwrite()`? WDYT @eliocamp

Reply:
That's a though one. Honestly this behaviour is kind of unexpected to me. I would've expected `fwrite` to either write an empty file or to throw an error. 

With `append == TRUE`, the file is left untouched, which would be the expected behaviour when writing an empty data.table, so returning the file would make sense, though.